### ISSUE TITLE
Section 2.2 missing

### ISSUE THREAD

**Issue by thebuckleyminer** [RunestoneInteractive/pythonds#106]
**What Course are you in**
pythonds

**What Page were you on**
https://runestone.academy/runestone/books/published/pythonds/index.html
https://runestone.academy/runestone/books/published/pythonds/ProperClasses/make_your_class_comparable.html

**Describe the bug**
Section 2.2 is just a header with no content. I tested two different browsers on two computers and section 2.2 appears to me missing. The only content that is displayed is the banner: "2.2. Making your Class Comparable".

It seems like all of the other chapters besides chapter two have more than two subsections as well, so I am wondering if somehow the last part of Chapter 2 got deleted off this specific server that is hosting it.

The link to the specific page is:
https://runestone.academy/runestone/books/published/pythonds/ProperClasses/make_your_class_comparable.html
The link to the table of contents is:
https://runestone.academy/runestone/books/published/pythonds/index.html

Maybe it is just me, or maybe there was not meant to be a section here. I just wanted to let you know just in case it was indeed a bug.

**Comment by bnmnetp** [comment]
The pythonds3 edition of this book has the content you are looking for.

**Comment by thebuckleyminer** [comment]
Sounds good!
I found it at: https://runestone.academy/runestone/books/published/pythonds3/index.html
I just figured since Chapter 2 did not end with: 
-.--. Summary
-.--. Key Terms
-.--. Discussion Questions
-.--. Programming Exercises
Like all eight of the other chapters do, that perhaps it was missing. But as you said, 2.2 is working under this link:
https://runestone.academy/runestone/books/published/pythonds3/index.html
Thanks for your help bnmnetp.
